modules/ztp-configuring-pgt-image-registry.adoc

```diff
@@ -0,0 +1,103 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-configuring-pgt-image-registry_{context}"]
+= Configuring the image registry using PolicyGenTemplate CRs
+
+You can use `PolicyGenTemplate` to apply to create the PV and PVC and patch `imageregistry` configuration. Select the appropriate `PolicyGenTemplate` for each `source-cr`. See Additional Resources for more help.
+
+.Prerequisites
+
+* You have installed and configured Zero Touch Provisioning (ZTP). For further information on this, see the topic on ZTP in Additional resources.
+
+.Procedure
+
+. Add the following YAML to the `StorageClass.yaml` file to configure the image registry:
++
+[source,yaml]
+----
+sourceFiles:
+   # storage class
+   - fileName: StorageClass.yaml
+     policyName: "sc-for-image-registry"
+     metadata:
+       name: image-registry-sc
+       annotations:
+         ran.openshift.io/ztp-deploy-wave: "100" <1>
+  # persistent volume claim
+  - fileName: StoragePVC.yaml
+    policyName: "pvc-for-image-registry"
+    metadata:
+      name: image-registry-pvc
+      namespace: openshift-image-registry
+        annotations:
+          ran.openshift.io/ztp-deploy-wave: "100"  <2>
+    spec:
+      accessModes:
+        ReadWriteMany
+      resources:
+        requests:
+          storage: 100Gi
+      storageClassName: image-registry-sc
+      volumeMode: Filesystem
+  # persistent volume
+  - fileName: ImageRegistryPV.yaml <3>
+    policyName: "pv-for-image-registry"
+    metadata:
+      annotations:
+        ran.openshift.io/ztp-deploy-wave: "100"  <4>
+  # image registry config
+  - fileName: ImageRegistryConfig.yaml <5>
+    policyName: "config-for-image-registry"
+    complianceType: musthave <5>
+    metadata:
+      annotations:
+        ran.openshift.io/ztp-deploy-wave: "100" <6>
+    spec:
+      storage:
+        pvc:
+          claim: "image-registry-pvc"
+----
+<1> Remove this when moved to the correct PGT (`site/group/common`).
+<2> Remove this when moved to the correct PGT (`site/group/common`).
+<3> This assumes that `mount_point` is set to `/var/imageregistry` in `SiteConfig` using StorageClass `image-registry-sc` (see the topic on configuring disk partitioning with `SiteConfig`).
+<4> Remove this when moved to the right PGT (`site/group/common`).
+<5> Configure registry to point to the PVC created above.
+<6> Remove this when moved to the right PGT (`site/group/common`)
+
+== Verifying the installation
```

Comment:
One heading per module. This needs to be `.Verification` like you have in `.Prerequisites` and `.Procedure`.